apps/frontend/src/app/2_molecules/WalletBalance/WalletBalance.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, useMemo } from 'react';
+
+import { SupportedTokenList, SupportedTokens } from '@sovryn/contracts';
+import { SimpleTable, SimpleTableRow } from '@sovryn/ui';
+
+import styles from './WalletBalance.module.css';
+
+export const WalletBalance: FC = () => {
+  const tokenOptions = useMemo(() => {
+    return SupportedTokenList.map(token => (
+      <SimpleTableRow
+        className={styles.row}
+        key={token.symbol}
+        label={token.symbol}
+        value="0" //TODO get balance from the wallet, task SOV-1101
+      />
+      //TODO: hardcoded, need to remove filter once we have a full list of tokens and ask Gilad to provide token symbols
+    )).filter(token => token.props.label !== SupportedTokens.sov);
```

Comment:
I'd do it the opposite way - create an array of the tokens we want to display above the component (e.g. `tokensToDisplay`) and check if a `token` is in the array. That way you won't need to adjust this filter every time we add a new token to `SupportedTokens`.